src/components/flowchart/styles/_node-icon.scss

```diff
@@ -0,0 +1,80 @@
+@import './variables';
+
+.node--icon {
+  circle,
+  rect {
```

Comment:
Do we have any control to add class names to these tags?

Reply:
We do, and I started on it, then changed my mind and left it as is, because it either ended up making things a bit more complicated on the JS end, or made the CSS rules more specific.

I'm not ruling out changing it to use classes instead at some point, but I didn't want to bite off any more than I already am in this PR, because it's big enough already.

Reply:
Cool cool something to consider in the future then 🙂